src/main/kotlin/org/quiltmc/community/modes/quilt/extensions/github/GithubExtension.kt

```diff
@@ -0,0 +1,152 @@
+package org.quiltmc.community.modes.quilt.extensions.github
+
+import com.expediagroup.graphql.client.ktor.GraphQLKtorClient
+import com.kotlindiscord.kord.extensions.DISCORD_RED
+import com.kotlindiscord.kord.extensions.checks.hasRole
+import com.kotlindiscord.kord.extensions.commands.Arguments
+import com.kotlindiscord.kord.extensions.commands.application.slash.ephemeralSubCommand
+import com.kotlindiscord.kord.extensions.commands.converters.impl.int
+import com.kotlindiscord.kord.extensions.commands.converters.impl.optionalString
+import com.kotlindiscord.kord.extensions.commands.converters.impl.string
+import com.kotlindiscord.kord.extensions.extensions.Extension
+import com.kotlindiscord.kord.extensions.extensions.ephemeralSlashCommand
+import com.kotlindiscord.kord.extensions.types.respond
+import dev.kord.core.behavior.channel.createEmbed
+import io.ktor.client.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.features.*
+import io.ktor.client.request.*
+import org.quiltmc.community.*
+import quilt.ghgen.DeleteIssue
+import quilt.ghgen.FindIssueId
+import quilt.ghgen.findissueid.*
+import java.net.URL
+
+class GithubExtension : Extension() {
+    override val name = "github"
+
+    private val client = GraphQLKtorClient(URL("https://api.github.com/graphql"),
+        HttpClient(engineFactory = CIO, block = {
+            defaultRequest {
+                header("Authorization", "bearer $GITHUB_TOKEN")
+            }
+        })
+    )
+
+    override suspend fun setup() {
+        for (guildId in GUILDS) {
+            ephemeralSlashCommand() {
+                name = "github"
+                description = "Perform privileged actions on the Quilt Github organization"
+
+                guild(guildId)
```

Comment:
Well now you've just gone and removed the permissions check entirely - anyone can run this now

Reply:
If you look not 10 lines down the check is in the subcommand, but for now I think it makes more sense to keep it at the top anyway.

Reply:
Oh, huh, I guess I did miss that. Checks are inherited though, so it's probably best at the top, yeah.

Reply:
Oh, huh, I guess I did miss that. Checks are inherited though, so it's probably best at the top, yeah.